pkg/commons-node/spec/which-spec.js

```diff
@@ -0,0 +1,68 @@
+'use babel';
+/* @flow */
+
+import which from '../which';
+
+describe('which', () => {
+  describe('on windows', () => {
+    const real_platform: string = process.platform;
+    const eol = '\r\n';
+    let os = require('os');
+    const real_eol = os.EOL;
+    beforeEach(() => {
+      Object.defineProperty(process, 'platform', {value: 'win32'});
+      os.EOL = eol;
+    });
+    afterEach(() => {
+      Object.defineProperty(process, 'platform', {value: real_platform});
+      os.EOL = real_eol;
+    });
+
+    it('calls where on Windows', () => {
+      let checkOutput: JasmineSpy = spyOn(require('../process'), 'checkOutput').andReturn('hello');
+      let param: string = '';
+      which(param);
+      expect(checkOutput).toHaveBeenCalledWith('where', [param]);
+    });
+
+    it('returns the first match', () => {
+      waitsForPromise(async () => {
+        let returnValue = {stdout: 'hello' + os.EOL + 'hello.exe' + os.EOL};
```

Comment:
It seems suspicious that the mocked return value is an object here and a simple string above. It looks like the actual return value isn't used above but still, it's easier to understand later if things are consistent.